test/unittest/layers/meson.build

```diff
@@ -62,7 +63,7 @@ test_target = [
   'unittest_layers_attention.cpp',
   'unittest_layers_dropout.cpp',
   'unittest_layers_reshape.cpp',
-  # 'unittest_layers_mol_attention.cpp',
+  'unittest_layers_mol_attention.cpp',
```

Comment:
Is mol_attention a molecular attention?
If so, does keras support it? (for gtest purpose)

Reply:
@skykongkong8 I don't think we need to support mol_attention layer anymore. This layer has been included in the main branch, so the unittest is also necessary for now. However, it would be nice to discuss removing this layer together.